app/stac_api/managers.py

```diff
@@ -153,6 +162,73 @@ def _parse_datetime_query(self, date_time):
             )
         return start, end
 
+    def filter_by_ids(self, ids_array):
+        '''Filter by ids parameter
```

Comment:
Maybe `Filter by item name` would be more explicit and won't introduced missunderstanding with the real id (pk)